backend/migrations/20240309000000-polls.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
```

Comment:
please rename the file to reflect the ≈date of the feature
e.g. 20250315...

migrations are applied in order